profiles/killbill/pom.xml

```diff
@@ -61,15 +61,25 @@
             <groupId>com.fasterxml.jackson.jaxrs</groupId>
             <artifactId>jackson-jaxrs-json-provider</artifactId>
         </dependency>
+        <!--
+        adding jsr305 to 'compile' help weird various guice errors at runtime
+        (at least in embedded mode when mvn jetty:run invoked). In fact, 'provided' scope to jsr305 make guice "think"
+        that we're not add @Nullable to various dependency, eg:
+
+        [Guice/NullInjectedIntoNonNullable]: null returned by binding at KillbillServerModule.configure(KillbillServerModule.java:125)
+          but the 1st parameter sqlLog of DBIProvider.setSqlLog(DBIProvider.java:97) is not @Nullable
+
+        in this example, DBIProvider.setSqlLog() is already annotated by @Nullable, but reported as "not @Nullable".
+        -->
         <dependency>
             <groupId>com.google.code.findbugs</groupId>
             <artifactId>jsr305</artifactId>
-            <scope>provided</scope>
+            <scope>compile</scope>
         </dependency>
         <dependency>
             <groupId>com.google.inject</groupId>
             <artifactId>guice</artifactId>
-            <scope>provided</scope>
+            <scope>compile</scope>
```

Comment:
That seems reasonable - who was 'providing' such package before this?

Reply:
FYI `compile` is the default scope (and it isn't overridden in the base pom.xml: https://github.com/killbill/killbill-oss-parent/blob/17b5048cafb6fe6c80a58ce12c6163844dea214c/pom.xml#L361-L365), so you shouldn't even need a scope at all.

Reply:
> That seems reasonable - who was 'providing' such package before this?

[I did](https://github.com/killbill/killbill/commit/429ac9b1166f724cbd18a953279e6400be3fa91d#diff-030ec92e9612718b2d2bc8aa5b7a65ebd296c063e121df61c273b3b00530b926) :man_facepalming: . I think in attempt to fix something else.

Reply:
> so you shouldn't even need a scope at all.

Fixed by this [commit](https://github.com/killbill/killbill/pull/1760/commits/aaedb5e4f3755ad87e74df93f946518e186d85e0).